src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -685,17 +685,10 @@ class Manager : public DoutPrefixProvider {
       for (const auto& queue_name : queues) {
         // try to lock the queue to check if it is owned by this rgw
         // or if ownership needs to be taken
-        librados::ObjectWriteOperation op;
-        op.assert_exists();
-        rados::cls::lock::lock(&op, queue_name+"_lock", 
-              ClsLockType::EXCLUSIVE,
-              lock_cookie, 
-              "" /*no tag*/,
-              "" /*no description*/,
-              failover_time,
-              LOCK_FLAG_MAY_RENEW);
-
-        ret = rgw_rados_operate(this, rados_ioctx, queue_name, &op, yield);
+
+        auto conn = rgw::redis::RGWRedis(io_context).get_conn();
```

Comment:
can probably use the same connection for all queues. please move outside of the loop